app/helpers/solidus_sale_prices/soft_delete.rb

```diff
@@ -9,5 +9,6 @@ module SoftDelete
       acts_as_paranoid
       include Discard::Model
       self.discard_column = :deleted_at
+    end
```

Comment:
I'd appreciate a pass to clean up the commit sequence, so changes like this are in the right commits.

Reply:
Done!